deploy/common/k8sbased.sh

```diff
@@ -613,17 +623,30 @@ function launch_sensor {
         )
       fi
 
+      local helm_chart="$k8s_dir/sensor-deploy/chart"
+
+      if [[ -n "${SENSOR_CHART_DIR_OVERRIDE}" ]]; then
+        helm_chart="${SENSOR_CHART_DIR_OVERRIDE}"
+      fi
+
       if [[ -n "$CI" ]]; then
-        helm lint "$k8s_dir/sensor-deploy/chart"
-        helm lint "$k8s_dir/sensor-deploy/chart" -n stackrox
-        helm lint "$k8s_dir/sensor-deploy/chart" -n stackrox "${helm_args[@]}" "${extra_helm_config[@]}"
+        helm lint "${helm_chart}"
+        helm lint "${helm_chart}" -n stackrox
+        helm lint "${helm_chart}" -n stackrox "${helm_args[@]}" "${extra_helm_config[@]}"
       fi
+
       if [[ "$sensor_namespace" != "stackrox" ]]; then
         kubectl create namespace "$sensor_namespace" &>/dev/null || true
         kubectl -n "$sensor_namespace" get secret stackrox &>/dev/null || kubectl -n "$sensor_namespace" create -f - < <("${common_dir}/pull-secret.sh" stackrox docker.io)
       fi
-      helm upgrade --install -n "$sensor_namespace" --create-namespace stackrox-secured-cluster-services "$k8s_dir/sensor-deploy/chart" \
+
+      set -x
+      echo "About to install sensor, running ls -la"
+      ls -la "${helm_chart}"
```

Comment:
Please don't forget to remove the debug output.

Reply:
Done